includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -7,14 +7,30 @@
 
 use WCPay\Exceptions\API_Exception;
 
-defined( 'ABSPATH' ) || exit;
+require_once WCPAY_ABSPATH . 'includes/in-person-payments/class-wc-payments-printed-receipt-sample-order.php';
```

Comment:
I'd suggest moving this after `defined( 'ABSPATH' ) || exit;`, that would make more sense.

Reply:
Done!